Firebase/Core/Private/FIRAppInternal.h

```diff
@@ -134,16 +134,16 @@ typedef NSString *_Nullable (^FIRAppGetUIDImplementation)(void);
  */
 + (BOOL)isDefaultAppConfigured;
 
-
 /**
  * Registers a given third-party library with the given version number to be reported for
  * analyitcs.
  *
  * @param library Name of the library
  * @param version Version of the library
  */
-+ (void)registerLibrary:(nonnull NSString *)library withVersion:(nonnull NSString *)version
-    NS_SWIFT_NAME(registerLibrary(_:version:));
++ (void)registerLibrary:(nonnull NSString *)library
+            withVersion:(nonnull NSString *)version NS_SWIFT_NAME(registerLibrary(_:version
```

Comment:
Unfortunately, clang-format can't handle `NS_SWIFT_NAME` annotations with more than one colon in them. Even more unfortunately, clang can't handle them when they're split across lines like this. As of now this is a broken build. The fix is to format this function declaration by hand and surround it with clang-format off/on, as done here:

https://github.com/firebase/firebase-ios-sdk/blob/90dedd3d024a2859d0cf514efbfc97e471891c18/Firestore/Source/Public/FIRQuery.h#L111

Reply:
Thanks for the tip! I was looking at this earlier and wonder what happened. I'll format it manually with the format comments.